cmd/vic-machine-server/Dockerfile

```diff
@@ -9,7 +9,7 @@ FROM photon:2.0
 
 RUN set -eux; \
      tdnf distro-sync --refresh -y; \
-     tdnf install shadow -y; \
+     tdnf install shadow-4.2.1-16.ph2.x86_64 -y; \
```

Comment:
Have you tested it?

Reply:
Yes. I  have tested locally and it uses the shadow version I specified to install. Part of log:

---> Running in bef335677dc6
+ tdnf distro-sync --refresh -y
Nothing to do.
Refreshing metadata for: 'VMware Photon Linux 2.0(x86_64) Updates'
Refreshing metadata for: 'VMware Photon Linux 2.0(x86_64)'
Refreshing metadata for: 'VMware Photon Extras 2.0(x86_64)'
+ tdnf install shadow-4.2.1-16.ph2.x86_64 -y

Installing:
Linux-PAM                x86_64       1.3.0-1.ph2      photon       912.55k 934452
cracklib                 x86_64       2.9.6-8.ph2      photon        44.55k 45616
shadow                   x86_64       4.2.1-16.ph2     photon-updates   1.68M 1764270
Total installed size:   2.62M 2744338

Downloading:



Testing transaction
Running transaction
Installing/Updating: cracklib-2.9.6-8.ph2.x86_64
using empty dict to provide pw_dict
Installing/Updating: Linux-PAM-1.3.0-1.ph2.x86_64
Installing/Updating: shadow-4.2.1-16.ph2.x86_64

Complete!
+ tdnf info installed
...

Reply:
shadow-4.2.1-16 is the version. Can you try it?

Reply:
Hi Daniel, tried below three ways. Seems shadow-4.2.1-16 not working but  shadow-4.2.1-16.ph2 or shadow-4.2.1-16.ph2.x86_64 looks good.

root [ / ]# tdnf install shadow-4.2.1-16.ph2 -y

Installing:
shadow                                              x86_64                    4.2.1-16.ph2                       photon-updates                1.68M 1764270

Total installed size:   1.68M 1764270

Downloading:
shadow                                  761060    100%
Testing transaction
Running transaction
Installing/Updating: shadow-4.2.1-16.ph2.x86_64

Complete!
root [ / ]# tdnf install shadow-4.2.1-16.ph2.x86_64 -y
Package shadow-4.2.1-16.ph2.x86_64 is already installed.
Nothing to do.
root [ / ]# tdnf install shadow-4.2.1-16  -y
Error(1013) : Packagelist was empty